src/jv.c

```diff
@@ -1013,6 +1018,7 @@ jv jv_array_concat(jv a, jv b) {
     a = jv_array_append(a, elem);
   }
   jv_free(b);
+  if(jv_is_borrowed(a)) return jv_unborrow(a);
```

Comment:
Superfluous.  The result of `jv_array_set()` on a borrowed array should always be unborrowed, right?